docs/workflow/testing/libraries/testing.md

```diff
@@ -7,59 +7,60 @@ To build the tests and run them you can call the libraries build script.
 **Examples**
 - The following shows how to build only the tests but not run them:
 ```
-build.cmd/sh -subset libs.tests
+build.cmd/sh libs.tests
 ```
 
 - The following builds and runs all tests in release configuration:
 ```
-build.cmd/sh -subset libs.tests -test -c Release
+build.cmd/sh libs.tests -test -c Release
 ```
 
 - The following example shows how to pass extra msbuild properties to ignore tests ignored in CI:
 ```
-build.cmd/sh -subset libs.tests -test /p:WithoutCategories=IgnoreForCI
+build.cmd/sh libs.tests -test /p:WithoutCategories=IgnoreForCI
 ```
 
 Unless you specifiy `-testnobuild`, test assemblies are implicitly built when invoking the `Test` action.
 - The following shows how to only test the libraries without building them
 ```
-build.cmd/sh -subset libs.tests -test -testnobuild
+build.cmd/sh -test -testnobuild
 ```
 
 ## Running tests on the command line
 
-To build tests you need to specify the `test` subset when invoking build.cmd/sh: `build.cmd/sh -subset libs.tests`.
+To build tests you need to specify the `libs.tests` subset when invoking build.cmd/sh: `build.cmd/sh libs.tests`.
 
-The easiest (and recommended) way to build and run the tests for a specific library, is to invoke the `Test` target on that library:
+The easiest (and recommended) way to build and run the tests for a specific library, is to invoke `dotnet test` on that library:
 ```cmd
-cd src\libraries\System.Collections.Immutable\tests
-dotnet build /t:Test
+dotnet test src\libraries\System.Collections.Immutable\tests
 ```
 
-It is possible to pass parameters to the underlying xunit runner via the `XUnitOptions` parameter, e.g.:
+It is possible to pass parameters to the underlying VSTest runner, e.g.:
 ```cmd
-dotnet build /t:Test /p:XUnitOptions="-class Test.ClassUnderTests"
+dotnet test --filter Test.ClassUnderTests
```

Comment:
Does this mean the wasm, iOS and Android runners will need to understand the `--filter` argument instead of the `-class` or `-method` argument? cc: @akoeplinger

Reply:
I don't think so. For mobile we still use xharness and just override the VSTest target so that `dotnet test` works. I don't see any easy way to make that VSTest specific filter syntax work on mobile.